### PR TITLE
Ingredients list

### PR DESCRIPTION
**Pull request by Sarahabidar**
Create a IngredientsList Component to display ingredients and flavors

### CODE REVIEW

File: components/IngredientsList.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  };
+  return (
+    <Container>
+      <Title>Ingredients Overview</Title>
+      <List>
+        {ingredients.map((ingredient) => (
+          <ListItem
+            key={ingredient._id}
+            flavor={ingredient.flavor}
+            color={flavorColors[ingredient.flavor]}
```
Comment (NikRoe) [inline components/IngredientsList.js]: when passing props to a styled component, you should prefix it with a dollar sign. You should also get this error in your terminal:

```
styled-components: it looks like an unknown prop "flavor" is being sent through to the DOM, which will likely trigger a React console error. If you would like automatic filtering of unknown props, you can opt-into that behavior via `<StyleSheetManager shouldForwardProp={...}>` (connect an API like `@emotion/is-prop-valid`) or consider using transient props (`$` prefix for automatic filtering.)
```

```suggestion
            $color={flavorColors[ingredient.flavor]}
```

File: components/IngredientsList.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  margin-bottom: 20px;
+`;
+const List = styled.ul`
+  list-style: none;
+  padding: 0;
+`;
+const ListItem = styled.li`
+  display: flex;
+  justify-content: space-between;
+  background-color: ${(props) => props.color || "#fff"};
```
Comment (NikRoe) [inline components/IngredientsList.js:81]:
```suggestion
  background-color: ${(props) => props.$color || "#fff"};
```

File: components/IngredientsList.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    Umami: "#188989",
+  };
+  return (
+    <Container>
+      <Title>Ingredients Overview</Title>
+      <List>
+        {ingredients.map((ingredient) => (
+          <ListItem
+            key={ingredient._id}
+            flavor={ingredient.flavor}
```
Comment (NikRoe) [inline components/IngredientsList.js]: do your use this prop?

If not, you can remove it

File: components/IngredientsList.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import ingredients from "@/assets/ingredients.json";
+//import flavor from "@/assets/flavors.json";
```
Comment (NikRoe) [inline components/IngredientsList.js:2]: You can remove this line

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi there,

nicely done!

I left some comments and suggestions.

Please take a look at those ✌️

File: components/IngredientsList.js (hunk 4 above)
Comment (Sarahabidar) [reply components/IngredientsList.js:2]: @NikRoe Thank you for the comments on our code. We will make the changes and get back to you.

**Pull request closed without merging**